src/main.rs

```diff
@@ -34,6 +34,8 @@ fn get_interface(specified_interface: &str) -> Option<Interface> {
 
 #[derive(Parser, Debug)]
 struct Args {
+    #[arg(long)]
+    identity: Option<String>,
```

Comment:
In reality, identity payloads are opaque byte blobs, but making the demo app take in a string that we use as-is (rather than, say, base64 decoding it or reading it from a file or whatever) made sense for the demo app.